SDLnetUDP.c

```diff
@@ -22,7 +22,7 @@
 #include "SDLnetsys.h"
 #include "SDL_net.h"
 
-#if defined(__WIN32__) || defined(__OS2__)
+#if defined(_WIN32) || defined(__OS2__)
```

Comment:
SDL2 defines `__WINRT__` for winrt.
```suggestion
#if defined(__WIN32__) || defined(__WINRT__) || defined(__OS2__)
```